wnprc_billing/src/org/labkey/wnprc_billing/dataentry/NonAnimalChargesFormSection.java

```diff
@@ -0,0 +1,27 @@
+package org.labkey.wnprc_billing.dataentry;
+
+import org.labkey.api.ehr.EHRService;
+import org.labkey.api.ehr.dataentry.SimpleFormSection;
+import org.labkey.api.view.template.ClientDependency;
+
+import java.util.Collections;
+
+/**
+ * Class to administer Ext4JS component/panel for ehr_billing.miscCharges data entry for charges without animal id and project
+ */
+public class NonAnimalChargesFormSection extends SimpleFormSection {
+
+    public NonAnimalChargesFormSection()
+    {
+        this(EHRService.FORM_SECTION_LOCATION.Body);
+    }
+
+    public NonAnimalChargesFormSection(EHRService.FORM_SECTION_LOCATION location)
+    {
+        super("ehr_billing", "miscCharges", "Misc. Charges", "ehr-gridpanel", location);
+        setConfigSources(Collections.singletonList("Task"));
+
+        addClientDependency(ClientDependency.fromPath("ehr_billing/form/field/EHRBillingRowObserverEntryField.js"));
```

Comment:
It looks like this dependency is already being added in the NonAnimalChargesFormType.java class below. Is it needed here?

Reply:
not needed here & removed. it was added because i was seeing some weird behavior, but missed to take it out after.